report/topology.go

```diff
@@ -113,6 +113,16 @@ func (t Topology) AddNode(node Node) Topology {
 	return t
 }
 
+// ReplaceNode adds node to the topology under key nodeID; if a
+// node already exists for this key, node replaces that node.
+// The same topology is returned to enable chaining.
```

Comment:
I've just removed the chaining in `AddNode()` in #3075, so it should b removed here too.

Reply:
Removed.